Eastern-Research-Group/lew
# Feature/switch to axios

## Related Issues:
* Closes #81 
* This PR depends on #82, so that one will need to be merged first

## Main Changes:
* Switched from `request-promise` to `axios` because `request-promise` has been deprecated due to `request` being deprecated

## Steps To Test:
1. Run `npm ci`
2. Run `npm run test`
3. Verify all tests pass
4. Run `npm run start`
5. Verify the app works
